index.js

```diff
@@ -194,24 +195,24 @@ function purgePlayers(client) {
 }
 
 function convertToByte(num) {
-	num = num & 0xFF;
-	num = num > 127 ? num - 256 : num;
-	return num;
+	num &= 0xFF;
+	new_num = num > 127 ? num - 256 : num;
+	return new_num;
 }
 
 function convertAngle(ang, num) {
-	ang = ang / 256 * Math.PI * 2;
-	ang = (((ang * -1) * RAD2DEG) - (num != undefined ? num : 0)) * 256 / 360;
-	return convertToByte(ang);
+	new_ang = ang / 256 * Math.PI * 2;
+	new_ang = (((ang * -1) * RAD2DEG) - (num != undefined ? num : 0)) * 256 / 360;
+	return convertToByte(new_ang);
 }
 
 function spawnEntity(packet, client) {
 	playerRotations[packet.id] = [convertAngle(packet.yaw, 180), convertAngle(packet.pitch)];
 	playerGamemodes[packet.id] = GAMEMODES[packet.gamemode ?? "survival"];
 	playerSkins[packet.id] = packet.cosmetics.skin;
-	if (playerUUIDs[packet.id] == undefined || playerPositions[packet.id] == undefined) return;
-	if (playerGamemodes[packet.id] == GAMEMODES.spectator) return;
-	if (players.includes(packet.id)) return;
+	if (playerUUIDs[packet.id] == undefined || playerPositions[packet.id] == undefined) {return;}
+	if (playerGamemodes[packet.id] == GAMEMODES.spectator) {return;}
+	if (players.includes(packet.id)) {return;}
```

Comment:
```suggestion
	if (playerUUIDs[packet.id] == undefined || playerPositions[packet.id] == undefined) return;
	if (playerGamemodes[packet.id] == GAMEMODES.spectator) return;
	if (players.includes(packet.id)) return;
```
NO MORE BARAKEETS!!!